src/app/docs/docs.html

```diff
@@ -0,0 +1,255 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```

Comment:
You can add description in your meta tag, Like author desc, page desc, details

Reply:
Added author and description meta data.